elastio-lambda-stream-backup/readme.md

```diff
@@ -0,0 +1,68 @@
+Before starting, please check do you have aws-cliv2, and docker cli installed.
+
+1. From AWS console, select IAM, select Roles then create IAM role with "AdministratorAccess".  
+
+2. From the AWS console, select ECR and create an ECR repository for storing your docker image.
+
+3. Unpack archive and go into unpacked folder.
+# if you want use arm64/amd64 architecture you should use the instance in the same architecture to building image
+
+4. Build and push docker image:
+	# Before starting doing this paragraph look at dockerfile and read comments,
+	# In the comments you can see the possible values for arguments such as ARCH(architecture), VERSION_TAG(elastio cli version)
+	4.1 Register your docker cli by getting the ecr public image with the command:
+	"aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws"
+
+	4.2. Open your ECR repository that we created earlier and press the option "View push commands"
+	If will show you 4 commands that you should run in your terminal that will build your docker image.
+
+5. Create a Lambda function:
+
+	5.1 Go to your AWS console and select the Lambda service, select Create Function, then select "Container image".
```

Comment:
Look into how to make numbered lists in Markdown.  You should not be doing the numbering manually.